scapy/arch/windows/__init__.py

```diff
@@ -986,7 +994,7 @@ def _get_valid_guid():
 def _route_add_loopback(routes=None, ipv6=False, iflist=None):
     """Add a route to 127.0.0.1 and ::1 to simplify unit tests on Windows"""
     if not WINDOWS:
-        warning("Not available")
+        warning("You have no idea what you are doing, stop it ! WINDOWS ONLY")
```

Comment:
That's a bit rough. How about "Calling _route_add_loopback() is only valid on a Windows platform!"

Reply:
Heh, I've seen people using that on stackoverflow.